content/md/first-steps/manage-reference-entities.md

```diff
@@ -0,0 +1,233 @@
+
+### Edit an image attribute
+  - *Max file size (MB):* maximum file size allowed in MB
+  - *Allowed extensions*: allowed extensions files (gif, jpeg, jpg, pdf, png...)
+
+### Edit a single/multiple options attribute  
+  - A `Manage options` button to [manage the options of the list](/articles/manage-reference-entities.html#manage-the-options)
+
+### Edit a reference entity simple/multiple link attribute    
+  - *Related reference entity*: the reference entity linked to this attribute, this information cannot be updated
+
+## Manage the options
+To define the options for a single/multiple options attribute, follow these steps:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Edit` icon for the attribute
+1.  An edit pop-in opens with the attribute parameters
+1.  Click on the `Manage options` button
+1.  A fullscreen opens to manage the options
+
+![image](../img/ReferenceEntity_ManageOptions.png)
+
+### Add options
+1. Write your option label
+1. The option code is automatically generated using the label but you can update it (this code must be unique and may only contain letters, numbers and underscores)
+1. Write your next option label
+1. Click on `Save` button, the fullscreen is closed
+
+![image](../img/ReferenceEntity_AddOptions.png)
+
+:::warning
+The code can't be edited afterwards, you can only edit/translate the label.  
+:::
+
+:::tips
+To quickly input all your options, you just have to input the 1st label, then press `Enter`, input the 2nd label, then press `Enter` and so on...
+:::
+
+:::info
+To guarantee that the PIM is functional and runs smoothly, we defined a volume limitation of 100 options per attribute for this feature. If you try to create more than 100 options, you will have a blocking message.  
+If you need to manage more than 100 options per attribute, we advise you to create a dedicated reference entity that can be linked to this entity.
+:::
+
+### Translate options
+1. Change the catalog locale using the switcher
+1. Translate the option label
+1. Translate the next option label
+1. Click on `Save` button, the fullscreen is closed
+
+![image](../img/ReferenceEntity_TranslateOptions.png)
+
+:::info
+To help you for the translation, the labels of the option on other locales are displayed in the right panel.
+:::
+
+:::tips
+To quickly translate all your options, you just have to translate the 1st label, then press `Enter`, translate the 2nd label, then press `Enter` and so on...
+:::
+
+
+### Remove options
+To remove an option, click on the cross at the end of the option line.
+
+## Define the completeness
+To define the attributes required for the completeness of a record:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Edit` icon for an attribute
+1.  Tick `Required for completeness` if this attribute is required for the completeness of a record
+
+![image](../img/ReferenceEntity_EditAttribute_Completeness.png)
```

Comment:
Peut être ajouter un focus sur l'image pour montrer ce qu'il faut cocher?

Reply:
J'ai mis juste la pop in